build.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+# cuDF build script
+
+# This script is used to build the component(s) in this repo from
+# source, and can be called with various options to customize the
+# build as needed (see the help output for details)
+
+# Abort script on first error
+set -e
+
+NUMARGS=$#
+ARGS=$*
+
+# NOTE: ensure all dir changes are relative to the location of this
+# script, and that this script resides in the repo dir!
+REPODIR=$(cd $(dirname $0); pwd)
+
+VALIDARGS="clean cuXfilter -v -g -n --allgpuarch -h"
+HELP="$0 [clean] [cuXfilter] [cuDatashader] [-v] [-g] [-n] [-h]
+   clean        - remove all existing build artifacts and configuration (start
+                  over)
+   cuXfilter    - build the cuXfilter library only
+   cuDatashader - build the cuDatashader library only
+   -v           - verbose build mode
+   -g           - build for debug
+   -n           - no install step
+   --allgpuarch - build for all supported GPU architectures
+   -h           - print this text
+"
+CUXFILTER_BUILD_DIR=${REPODIR}/python/cuXfilter/build
```

Comment:
All lower case directories as well. Should change the python folder name to all lower in the repo as well.